index.html

```diff
@@ -32,25 +32,20 @@
     <aside class="mdc-drawer mdc-drawer--dismissible">
       <div class="mdc-drawer__content">
         <h2 class="mdc-top-app-bar__title">Topics</h2>
-        <ul class="mdc-list">
-          <li>
-            <a class="mdc-list-item mdc-list-item--activated" href="#" aria-selected="true">
-              <i class="material-icons mdc-list-item__graphic" aria-hidden="true">grade</i>
-              <span class="mdc-list-item__text">The Keyword</span>
-            </a>
-          </li>
+        <nav class="mdc-list">
+          <a class="mdc-list-item mdc-list-item--activated" href="#" aria-selected="true">
```

Comment:
Would you mind elaborating on why a list cannot be used here? After all this is a list of items..

Reply:
Our [prescribed HTML structure](https://github.com/material-components/material-components-web/tree/master/packages/mdc-drawer#html-structure) is to use a `<nav>` in the drawer for accessibility. According to [spec](https://www.w3.org/TR/html-aria/#index-aria-navigation), it is "a collection of navigational elements (usually links) for navigating the document or related documents."

`<ul role="navigation">` would also work, but it's a bit less semantic than a straight `<nav>`.

Reply:
Thanks for shedding some light on this. I'm surprised, because as far as I know common practice is to have a list of links that is used within `<nav>` marked up as a list. That way you do not have to use ARIA roles (the spec you linked).

The examples [on MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) as well as the [WHATWG spec samples](https://html.spec.whatwg.org/multipage/sections.html#the-nav-element) use lists.

Is MDC mandating that one just dumps the links directly into `<nav>` and cannot use a list inside?

Reply:
Thanks for the links! I'm not knowledgeable enough to give a good answer, do you want to submit a bug or yaqs question to get clarification?